forge-gui/res/cardsfolder/upcoming/patchwork_beastie.txt

```diff
@@ -2,9 +2,10 @@ Name:Patchwork Beastie
 ManaCost:G
 Types:Artifact Creature Beast
 PT:3/3
-S:Mode$ Continuous | Affected$ Card.Self | AddHiddenKeyword$ CARDNAME can't attack or block. | Condition$ Delirium | Delirium — CARDNAME can't attack or block unless there are four or more card types among cards in your graveyard.
+S:Mode$ Continuous | Affected$ Card.Self | AddHiddenKeyword$ CARDNAME can't attack or block. | CheckSVar$ X | SVarCompare$ LT4 | Delirium — CARDNAME can't attack or block unless there are four or more card types among cards in your graveyard.

```

Comment:
why revert?

Reply:
Because it should be unable to attack or block when the delirium is NOT satisfied.

Reply:
haha my bad
but you can use `Count$Delirium.0.1`
and it still needs Description$ :p